classes.py

```diff
@@ -0,0 +1,26 @@
+# class todo_status:
```

Comment:
Commented out code shouldn't be merged to master... you could leave it on some branch if you want to keep it around.

Also, this shouldn't really be called classes.py - similar to keeping functions and classes small, it's nice to keep files "small".
In most languages you'd do one file per class. The python community is a bit divided but I'd still say one file per class is a good rule of thumb but you can group closely related classes together if you want.